src/components/Users.js

```diff
@@ -0,0 +1,76 @@
+import { useEffect, useState } from "react";
+import { BaseUrl } from "../Constants";
+import axios from "axios";
+import localStorageService from "../LocalStorageService";
+import Table from "react-bootstrap/Table";
+import { Spinner } from "react-bootstrap";
+
+export default function Users() {
+  const [users, setUsers] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    const token = localStorageService.get("auth-token");
```

Comment:
Hi Param, we need to set the loading to true when the request starts processing like this `setIsLoading(true)`